core/src/main/java/uk/gov/onelogin/core/navigation/data/SignOutRoutes.kt

```diff
@@ -13,7 +13,7 @@ sealed class SignOutRoutes(private val route: String) : NavRoute {
 
     data object Success : SignOutRoutes("sign_out/success")
 
-    data object SignOutError : ErrorRoutes("/sign_out/error")
+    data object SignOutWalletError : ErrorRoutes("/sign_out/error")
```

Comment:
Same as above, probably change the route too
```suggestion
    data object SignOutWalletError : ErrorRoutes("/sign_out/wallet_error")
```